src/BucketRateLimiter.sol

```diff
@@ -16,8 +16,8 @@ contract BucketRateLimiter is IRateLimiter, IPausable, Initializable, PausableUp
     BucketLimiter.Limit public limit;
```

Comment:
I didn't create an admin role for the `BucketRateLimiter` bc it doesn't expose any admin functionality